test/ProgrammaticIncentives.t.sol

```diff
@@ -286,7 +286,15 @@ contract ProgrammaticIncentivesTests is BytecodeConstants, Test {
         uint256 newCooldownHorizon =
             ((block.timestamp - configuration.startTime) / configuration.cooldownSeconds + 1) * configuration.cooldownSeconds
             + configuration.startTime;
-        emit ButtonPressed(address(this), newCooldownHorizon);
+        if (block.chainid == 17000
+            && (address(tokenHopper) == 0x2B35f55ad87900Db6479AEdD9eB2c53C58cf85d9 ||
```

Comment:
can we get rid of handling both of these paths?

Reply:
yes, only needed this so `forge t -vvv` would pass for testnet deployments. I can delete know 🤷

Reply:
this logic was deleted in https://github.com/Layr-Labs/EigenHopper/pull/8/commits/cc4172464d2ffd81b9f1acf9d4fe0adf0b64fea1